ql/src/java/org/apache/hadoop/hive/ql/ddl/table/constraint/add/AlterTableAddConstraintAnalyzer.java

```diff
@@ -79,7 +79,8 @@ protected void analyzeCommand(TableName tableName, Map<String, String> partition
       throw new SemanticException(ErrorMsg.NOT_RECOGNIZED_CONSTRAINT.getMsg(constraintNode.getToken().getText()));
     }
 
-    Constraints constraints = new Constraints(primaryKeys, foreignKeys, null, uniqueConstraints, null, checkConstraints);
+    Constraints constraints =
+        new Constraints(primaryKeys, foreignKeys, null, uniqueConstraints, null, checkConstraints);
```

Comment:
All changes in this file are out of scope of the PR.  Please revert.

Reply:
Removed.